Platform.md

```diff
@@ -53,11 +51,11 @@ This file contains instructions for installing dependencies on different platfor
        /usr/local/Cellar/openssl/1.0.2l (1,709 files, 12.1MB)
          Poured from bottle on 2017-05-31 at 15:32:30`
       Use the latest stable 1.0.2 version available, which is 1.0.2l here   
-    * Run `brew switch openssl <latest stable 1.0.2 version>`. If the version is already installed, the path for this version of OpenSSL will be displayed.
+    * Run `brew switch openssl <latest stable 1.1.0 version>`. If the version is already installed, the path for this version of OpenSSL will be displayed.
       Example output:
       `Cleaning /usr/local/Cellar/openssl/1.0.2l
-       Opt link created for /usr/local/Cellar/openssl/1.0.2l`
-      Use the path (`/usr/local/Cellar/openssl/1.0.2l` in the example above) in your <BASE_SDK_DIRECTORY>/network/CMakeLists.txt 
+       Opt link created for /usr/local/Cellar/openssl/1.1.0l`
```

Comment:
Is this correct?

Reply:
It was not correct but I modified it so it points to the correct location - at least on my machine.